Test error with numpy >= 1.25

The function `run_module_suite` has been removed from  numpy >= 1.25.

This is causing the following error
```
ImportError while importing test module 'venv/lib/python3.11/site-packages/astroML/tests/test_resample.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
astroML/tests/test_resample.py:2: in <module>
    from numpy.testing import assert_allclose, run_module_suite
E   ImportError: cannot import name 'run_module_suite' from 'numpy.testing' (venv/lib/python3.11/site-packages/numpy/testing/__init__.py)
```

--- a/astroML/tests/test_resample.py
+++ b/astroML/tests/test_resample.py
@@ -1,5 +1,5 @@
 import numpy as np
-from numpy.testing import assert_allclose, run_module_suite
+from numpy.testing import assert_allclose
 
 from astroML.resample import bootstrap, jackknife
 from astroML.stats import mean_sigma
@@ -87,7 +87,3 @@ def test_jackknife_pass_indices():
                      pass_indices=True)
 
     assert_allclose(res1, res2)
-
-
-if __name__ == '__main__':
-    run_module_suite()

